### PR TITLE
 Remove __TABLE__ and fixed spark xml version

### PR DESCRIPTION
**Pull request by badrinathpatchikolla**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -18,8 +18,9 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-sql" % sparkVersion % "provided",
   "org.apache.spark" %% "spark-sql-kafka-0-10" % sparkVersion % "provided" excludeAll(ExclusionRule(organization = "net.jpountz.lz4")),
   "org.apache.spark" %% "spark-avro" % sparkVersion,
-  "com.databricks" %% "spark-xml" % "0.10.0",
+  "com.databricks" %% "spark-xml" % "0.14.0",
   "com.github.music-of-the-ainur" %% "quenya-dsl" % s"1.2.0-${majorVersion}",
+  "io.netty" % "netty-all" % "4.1.71.Final",
```
Comment (mantovani) [inline build.sbt:23]: Why did you add this ?

Comment (badrinathpatchikolla) [reply build.sbt:23]: This dependency is present in 3.2 branch ,should I remove it?

Comment (badrinathpatchikolla) [reply build.sbt]: Removed the Dependency
